.github/workflows/update-synced-issue.yml

```diff
@@ -0,0 +1,35 @@
+name: Update GitHub Issue synced on Jira
+
+on:
+  issue_comment:
+
+jobs:
+  sync-issue:
+    if: secrets.JIRA_BASE_URL != '' && secrets.JIRA_USER_EMAIL != '' && secrets.JIRA_API_TOKEN != ''
+    name: Sync GitHub Issue to Jira
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if issue is synced
+        uses: actions/github-script@v7
+        id: check_label
+        with:
+          script: |
+            const labels = context.payload.issue.labels.map(label => label.name);
+            const hasLabel = labels.includes('sync-to-jira');
+            core.setOutput('has-label', hasLabel);
+      - name: Setup python
+        if: steps.check_label.outputs.has-label == 'true'
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          cache: "pip" # caching pip dependencies
+
+      - name: Install python requirements
+        if: steps.check_label.outputs.has-label == 'true'
+        run: |
+          pip install --upgrade pip && pip install -r scripts/requirements.txt
+
+      - name: Run update script
+        if: steps.check_label.outputs.has-label == 'true'
+        run: |
+          python3 jira_driver.py --issue ${{github.event.issue.number}} update --summary "Issue comment ${{github.event.action}}: ${{github.event.comment}}""
```

Comment:
We should assign the github substitutions like `${{github.event.action}}` to environment variables, otherwise we could have injection problems.